### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mchirino89**
It's just a tiny improvement to the README.md.

See if it makes sense to include it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +66,7 @@ swift create-xcframework --platform ios --platform macos ...
 
 #### Catalyst
 
-You can build your XCFrameworks with support for Mac Catalyst by specifying `--platform maccatalyst` on the command line. As you can't include or exclude Catalyst support in your `Package.swift` we don't try to build it automatically.
+You can build your `XCFrameworks` with support for Mac Catalyst by specifying `--platform maccatalyst` on the command line. As you can't include or exclude Catalyst support in your `Package.swift` we don't try to build it automatically.
```
Comment (bok-) [inline README.md:69]: XCFrameworks is a name of the product not a file name or code reference so I don't think the backticks make sense here, and they're inconsistent with the rest of the document.

**Review by bok- — COMMENTED** [review]
One minor comment about inconsistent backticks, everything else looks fine though so happy to approve once these are fixed.

File: README.md (hunk 1 above)
Comment (mchirino89) [reply README.md:69]: Got it, changing it

Comment (mchirino89) [inline README.md:69]:
```suggestion
You can build your XCFrameworks with support for Mac Catalyst by specifying `--platform maccatalyst` on the command line. As you can't include or exclude Catalyst support in your `Package.swift` we don't try to build it automatically.
```

Comment (mchirino89) [reply README.md]: Done

**Comment by mchirino89** [comment]
> One minor comment about inconsistent backticks, everything else looks fine though so happy to approve once these are fixed.

Done!

**Comment by mchirino89** [comment]
> > One minor comment about inconsistent backticks, everything else looks fine though so happy to approve once these are fixed.
> 
> Done!

Shall I close this @bok-?

**Comment by mchirino89** [comment]
> This PR has been marked stale as it has not been updated in 60 days. It will be closed in one week. It can be re-opened at any time.

Shall I close this? @bok-

**Pull request closed without merging**